<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>p2p</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resource/logo.ico</normaloff>resource/logo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>华文细黑</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>p2p下载器</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_m3u8">
     <property name="text">
      <string>m3u8</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_reptiles">
     <property name="text">
      <string>网页捕获音视频</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_mp3">
     <property name="text">
      <string>mp3播放器</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>torrent文件下载</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
